integrations/June/browser.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable */
```

Comment:
Why are we disabling linting for the whole file?

Reply:
removed the `eslint-disable` for whole file.